Assets/Character/Scripts/PlayerInput.cs

```diff
@@ -4,44 +4,44 @@ public static class PlayerInput
 {
     public static bool PressedRoll()
     {
-        return Input.GetKeyDown(KeyCode.C);
+        return Input.GetKeyDown(KeyCode.C) || Input.GetButtonDown("Roll");

```

Comment:
isn't this what this class is already doing? abstracting the button presses into one function? you don't need to abstract another level